webapp/channels/src/components/admin_console/ip_filtering/save_confirmation_modal.test.tsx

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {render, fireEvent} from '@testing-library/react';
+import React from 'react';
+
+import SaveConfirmationModal from './save_confirmation_modal';
+
+jest.mock('components/external_link', () => {
```

Comment:
Why mock the `ExternalLink` component? If it's something to do with missing context, you can use `renderWithFullContext` from `test/react_testing_utils` instead of using React Testing Library directly

Reply:
This is just for a shallow render. The `ExternalLink` component is redux connected to grab things like server ID/user ID/etc but I don't think having that fully qualified component in here adds any value to the test and adds a bunch of bloat in mock state

Reply:
You might be able to get away without that if you use `renderWithFullContext` here instead of `renderWithIntl`, but since our discussion on Friday got me to rename `renderWithFullContext` and use it everywhere (https://github.com/mattermost/mattermost/pull/25217), you don't need to bother changing this now. I'll make sure to hold that PR until this one goes in too since it'll definitely conflict with this